### PR TITLE
[Retrieval Docs] Update retrieval docs

### PR DESCRIPTION
**Pull request by w5688414**

### CODE REVIEW

File: docs/modules/retrieval.md [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
+2. 定义百中Search
+
+```
+baizhong_db = BaizhongSearch(knowledge_base_name="BaizhongExample")
+```
+
+3. 搜索
+
+```
+baizhong_db.search('百度今天的股价是多少？')
```
Comment (sijunhe) [inline docs/modules/retrieval.md]: 这个不对呀， 开发者跑这几行代码就能检索出东西吗？库里都没有东西

File: erniebot-agent/src/erniebot_agent/retrieval/baizhong_search.py [hunk 2]
```diff
@@ -42,7 +42,14 @@ def __init__(
...
         """
         self.base_url = os.getenv("AISTUDIO_BASE_URL", self._AISTUDIO_BASE_URL)
-        self.access_token = access_token
+        global_access_token = os.getenv("EB_AGENT_ACCESS_TOKEN")
+        if global_access_token is not None:
+            self.access_token = global_access_token
+        elif access_token is not None:
+            self.access_token = access_token
+        else:
+            raise BaizhongError("You must provide either a access_token or EB_AGENT_ACCESS_TOKEN.")
```
Comment (sijunhe) [inline erniebot-agent/src/erniebot_agent/retrieval/baizhong_search.py]: 和Model, File，Tool保持一致，统一用这个函数
https://github.com/PaddlePaddle/ERNIE-Bot-SDK/blob/develop/erniebot-agent/src/erniebot_agent/tools/remote_toolkit.py#L179

Comment (sijunhe) [inline erniebot-agent/src/erniebot_agent/retrieval/baizhong_search.py]: Please ensure that either an access_token is provided or the EB_AGENT_ACCESS_TOKEN is set up as an environment variable.

File: docs/modules/retrieval.md (hunk 1 above)
Comment (w5688414) [reply docs/modules/retrieval.md]: 第二步改成：连接AIStudio知识库 了

**Review by sijunhe — APPROVED** [review]

**Pull request merged**